### PR TITLE
Bar 112 implement scaffold

### PR DESCRIPTION
**Pull request by alex-medvedev-msc**
1. Simulated `quadratic` dataset with underlying linear model 
2. Implemented scaffold strategy
3. FLClient Callback API
4. Weight and grad logging to mlflow for scaffold monitoring
5. Plot loss landscape for `quadratic` dataset

### CODE REVIEW

File: src/fl/node_process.py [hunk 1]
```diff
@@ -98,30 +97,49 @@ def _train_model(self, client: FLClient) -> bool:
...
+                time.sleep(20)
                 continue
             except Exception as e:
+                print(e)
                 self.logger.error(e)
-                self.logger.error(e.with_traceback())
                 raise e
         return False
 
+    def create_callbacks(self):
```
Comment (dkolobok) [inline src/fl/node_process.py:116]: docs? not sure what callbacks for FL are. As well as landscape

File: src/local/experiment.py [hunk 2]
```diff
@@ -284,6 +288,99 @@ def load_best_model(self):
                 scheduler_params=self.cfg.experiment.scheduler
             )
 
+class QuadraticNNExperiment(NNExperiment):
```
Comment (dkolobok) [inline src/local/experiment.py:291]: could you describe what quadratic means?

File: src/local/utils.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
+from nn.models import BaseNet
+import numpy
+import itertools
+import plotly.graph_objects as go
+from sklearn.metrics import mean_squared_error
+import mlflow
+
+
+def plot_loss_landscape(model: BaseNet, x: numpy.ndarray, y: numpy.ndarray, beta: numpy.ndarray):
```
Comment (dkolobok) [inline src/local/utils.py:9]: docs for these functions? For instance, not clear what beta is
